### PR TITLE
Add podspec presubmit tests.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

**Comment by granluo** [comment]
This is to set FirebaseDatabase.podspec for presubmit and update post submit to let all changed podspecs from master head can be pushed to [firebase/specstesting ](https://github.com/firebase/SpecsTesting) repo

File: FirebaseDatabase.podspec [hunk 1]
```diff
@@ -88,3 +88,97 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
     int_tests.resources = 'FirebaseDatabase/Tests/Resources/GoogleService-Info.plist'
   end
 end
+
```
Comment (paulb777) [inline FirebaseDatabase.podspec:91]: Why the changes in this file?

File: .github/workflows/database.yml [hunk 2]
```diff
@@ -137,3 +137,17 @@ jobs:
...
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        target: [ios, tvos, macos, watchos]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Build and test
+      run: scripts/third_party/travis/retry.sh pod spec lint FirebaseDatabase.podspec --skip-tests --platforms=${{ matrix.target }} --sources='https://github.com/firebase/SpecsTesting','https://github.com/firebase/SpecsDev.git','https://github.com/firebase/SpecsStaging.git','https://cdn.cocoapods.org/'
```
Comment (paulb777) [inline .github/workflows/database.yml:153]: I don't understand the reason for only `FirebaseDatabase.podspec` here? Is it to verify with FirebaseDatabase first before adding the others?

File: .github/workflows/prerelease.yml [hunk 3]
```diff
@@ -2,9 +2,6 @@ name: prerelease
 
 on:
   pull_request:
-    paths:
-    - '.github/workflows/prerelease.yml'
```
Comment (paulb777) [inline .github/workflows/prerelease.yml:6]: Why are we deleting the paths here? Isn't it still useful to trigger the run when prerelease.yml changes?

Comment (granluo) [reply .github/workflows/prerelease.yml:6]: The workflow might  erase the testing podspecs repo and push all podspecs. This process takes three hours and  might fail presubmit tests.

File: .github/workflows/database.yml (hunk 2 above)
Comment (granluo) [reply .github/workflows/database.yml:153]: This is to set up database first and expand to other SDKs in another PR.

File: FirebaseDatabase.podspec (hunk 1 above)
Comment (granluo) [reply FirebaseDatabase.podspec]: Have reverted it

**Review by paulb777 — APPROVED** [review]
